### PR TITLE
Attitude field added to blackbox fields list

### PR DESCRIPTION
**Pull request by demvlad**
Improvement for ["Added flight IMU pitch, roll, yaw attitudes to blackbox log"](https://github.com/betaflight/betaflight/pull/14020) PR
Added "Attitude" field in blackbox log list
confatt

### CODE REVIEW

File: src/js/debug.js [hunk 1]
```diff
@@ -786,6 +786,7 @@ const DEBUG = {
         { text: "Altitude" },
         { text: "RSSI" },
         { text: "Gyro" },
+        { text: "Attitude" },
```
Comment (haslinghuis) [inline src/js/debug.js:789]:
```suggestion
```

For backwards compatibility: add attitude around line 816/817 (within semver block):
```
DEBUG.enableFields.splice(DEBUG.enableFields.indexOf("Gyro"), 0, "Attitude");
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Please rebase

File: src/js/debug.js (hunk 1 above)
Comment (demvlad) [reply src/js/debug.js:777]: The lines sequence are changed after rebase.
Can not understand, what i can do..

**Comment by haslinghuis** [comment]
@nerdCopter how would you approach ?

Would suggest using `git rebase -i HEAD~60` and pick first commit and drop all others.
And then probably `git pull --rebase upstream master`

**Comment by nerdCopter** [comment]
_sometimes_ i tend to 
```
git fetch --all
git merge upstream/master
git rebase -i upstream/master
```
this seems to keep my orderd commits, while rebasing upon master.

when i play with this stuff, i tend to make new test-branches try the commands, then wither
- redo it in original branch, or
- delete the original branch and rename the test-branch

**Comment by demvlad** [comment]
The one code row changed only. Rebased master and created push force new branch

File: src/js/debug.js [hunk 2]
```diff
@@ -774,6 +774,7 @@ const DEBUG = {
         "Altitude",
         "RSSI",
         "Gyro",
+        "Attitude",
```
Comment (haslinghuis) [inline src/js/debug.js:777]:
```suggestion
```

For backwards compatibility: add attitude after line 836 (within semver block):
```
DEBUG.enableFields.splice(DEBUG.enableFields.indexOf("Gyro"), 0, "Attitude");
```

**Review by haslinghuis — APPROVED** [review]

File: src/js/debug.js (hunk 2 above)
Comment (demvlad) [reply src/js/debug.js]: I've suspected, that this improvement can not be too easy :)

**Comment by demvlad** [comment]
The test result of BF, Configurator and explorer work are [showed this](https://github.com/betaflight/betaflight/pull/14020)

**Pull request merged**